src/js/react-exports.js

```diff
@@ -0,0 +1,162 @@
+// First import the withReact integration helper function
+import withReact from './with-react'
```

Comment:
Do you agree with single file, named exports and omitting polyfills (`promise` and `custom-elements-v-1`)?